packages/openapi-generator/src/generator-utils.ts

```diff
@@ -0,0 +1,6 @@
+import { readFileSync } from 'fs';
+import { GeneratorOptions } from './options';
+
+export function parseOptionsFromConfig(path: string): GeneratorOptions {
+  return JSON.parse(readFileSync(path, 'utf8'));
```

Comment:
As currently, we are not able to use config and command line options, I guess we should move the validation `if (options.input === '' || options.outputDir === '')` to here, as it returns `GeneratorOptions` that says `input` is not nullable.

Reply:
I think it's better if the check stays in `generateWithParsedOptions`, since `input` and `outputDir` are no longer required, it is possible to use no config but also not use `input  & `outputDir`, with this guard, both possiblities are checked. 🤔

Reply:
Good point.
Then this will be a point that `yargs` does better.
- When using config, we cannot mark `input` as required with `oclif` for the current implementation.
- We can still mark `inputDir` as required with `yargs` even using config.

Reply:
Agreed!